main.py

```diff
@@ -11,19 +12,42 @@ def create_app():
     CORS(app)
 
     @app.route('/movies', methods=['GET'])
-    def main_screen():
+    def search_movie():
         if request.method == 'GET':
-            search_value = request.args.get('search_value')
-            movies = filter_movie_short(search_movies_by_title(search_value))
-            return json_response(movies)
+            search_value = request.args.get('value')
+            movies = search_movies_by_title(search_value)
+            if movies['Response'] == 'False':
+                return json_response([])
```

Comment:
Handle error appropriately

Reply:
returns empty list since there is not exists movies with the query name.. why I need o handle it differently ?